### PR TITLE
DevWs for frient containing containing Zigbee Metering Plug

### PR DESCRIPTION
**Pull request by frient-design**

### CODE REVIEW

File: devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy [hunk 1]
```diff
@@ -1,184 +1,189 @@
...
+        fingerprint profileId: "0104", inClusters: "0000, 0002, 0003, 0006, 0702, 0B04", outClusters: "0003, 0019", manufacturer: "DAWON_DNS", model: "PM-B540-ZB",  deviceJoinName: "Dawon Outlet" // DAWON DNS Smart Plug
+        fingerprint profileId: "0104", inClusters: "0000, 0002, 0003, 0006, 0702, 0B04", outClusters: "0003, 0019", manufacturer: "DAWON_DNS", model: "ST-B550-ZB",  deviceJoinName: "Dawon Outlet" // DAWON DNS Smart Plug
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0702, 0B04", outClusters: "0003, 0019", manufacturer: "DAWON_DNS", model: "PM-C150-ZB",  deviceJoinName: "Dawon Outlet" // DAWON DNS In-Wall Outlet
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0702, 0B04", outClusters: "0003, 0019", manufacturer: "DAWON_DNS", model: "PM-C250-ZB",  deviceJoinName: "Dawon Outlet" // DAWON DNS In-Wall Outlet
+        fingerprint profileId: "0104", manufacturer: "frient A/S", model: "SPLZB-131",  deviceJoinName: "frient Smart Plug Mini" // frient smart plug mini, raw description: 02 0104 0051 10 09 0000 0702 0003 0009 0B04 0006 0004 0005 0002 05 0000 0019 000A 0003 0406
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy]: Please change deviceJoinName to `frient Outlet` (this applies to each device you've submitted)

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
@frient-design please adjust code in your request so diff here in PR would get fixed (I'm not sure what might be wrong, maybe it's encoding)

File: devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy (hunk 1 above)
Comment (rboy1) [reply devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy]: Fixed

**Comment by rboy1** [comment]
Unable to figure out why it's creating such a large diff. Repasted the original code from GitHub and manually modified the fingerprints but it's still generating a large diff.

**Comment by rboy1** [comment]
If it's not possible to fix the diff issue maybe @frient-design can delete the file and resubmit it

**Comment by frient-design** [comment]
I am deleting the file and resubmitting again

**Pull request closed without merging**